pkg/kubectl/cmd/scale.go

```diff
@@ -98,16 +105,13 @@ func RunScale(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []stri
 	if err != nil {
 		return err
 	}
-	mapping, err := r.ResourceMapping()
-	if err != nil {
-		return err
-	}
 
 	infos, err := r.Infos()
 	if err != nil {
 		return err
 	}
-
+	info := infos[0]
+	mapping := info.ResourceMapping()
```

Comment:
Using the old way of retrieving mapping returned an error "a resource mapping could not be loaded from *resource.FileVisitor".